<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBox</class>
 <widget class="QToolBox" name="ToolBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ToolBox</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsrc.qrc">
    <normaloff>:/icons/resources/icons/app-icons/logo_small.png</normaloff>:/icons/resources/icons/app-icons/logo_small.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string>0</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="currentIndex">
   <number>6</number>
  </property>
  <widget class="QWidget" name="drawingPage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>283</width>
     <height>351</height>
    </rect>
   </property>
   <attribute name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/app-icons/property.png</normaloff>:/icons/resources/icons/app-icons/property.png</iconset>
   </attribute>
   <attribute name="label">
    <string>Drawing Properties</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea_6">
      <property name="styleSheet">
       <string notr="true">QScrollArea::title { 
    background-color: transparent;
     subcontrol-position: top left; /* position at the top left*/ 
     padding:2 13px;
 } 

QScrollArea { 
     border: 2px solid gray;
     background-color: lightgray;
     border-radius: 5px; 
 } 
</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>277</width>
         <height>345</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="DrawingPropertiesWidget" name="drawingPropertiesWidget">
          <property name="toolTip">
           <string>Custom widget</string>
          </property>
          <property name="whatsThis">
           <string>You don't care</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="strokeStylePage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>283</width>
     <height>351</height>
    </rect>
   </property>
   <attribute name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/tool-icons/drawings.png</normaloff>:/icons/resources/icons/tool-icons/drawings.png</iconset>
   </attribute>
   <attribute name="label">
    <string>Stroke Style</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>235</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollArea::title { 
    background-color: transparent;
     subcontrol-position: top left; /* position at the top left*/ 
     padding:2 13px;
 } 

QScrollArea { 
     border: 2px solid gray;
     background-color: lightgray;
     border-radius: 5px; 
 } </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>277</width>
         <height>345</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>181</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="StrokeStyleWidget" name="strokeStyleWidget">
          <property name="toolTip">
           <string>Custom widget</string>
          </property>
          <property name="whatsThis">
           <string>You don't care</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="strokeColorPage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>283</width>
     <height>351</height>
    </rect>
   </property>
   <attribute name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/tool-icons/color1.png</normaloff>:/icons/resources/icons/tool-icons/color1.png</iconset>
   </attribute>
   <attribute name="label">
    <string>Stroke Color</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="styleSheet">
       <string notr="true">QScrollArea::title { 
    background-color: transparent;
     subcontrol-position: top left; /* position at the top left*/ 
     padding:2 13px;
 } 

QScrollArea { 
     border: 2px solid gray;
     background-color: lightgray;
     border-radius: 5px; 
 } </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>277</width>
         <height>345</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="ColorWidget" name="strokeColor">
            <property name="toolTip">
             <string>Custom color widget</string>
            </property>
            <property name="whatsThis">
             <string>You don't care</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="horizontalSpacing">
             <number>22</number>
            </property>
            <property name="verticalSpacing">
             <number>9</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Current Color:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="strokeColorLabel">
              <property name="styleSheet">
               <string notr="true">QLabel {
	border: 1px solid 
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Stroke Enabled:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="shouldStrokeCheckbox">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="strokeFillPage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>283</width>
     <height>351</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <attribute name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/tool-icons/color1.png</normaloff>:/icons/resources/icons/tool-icons/color1.png</iconset>
   </attribute>
   <attribute name="label">
    <string>Fill Color</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea_3">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollArea::title { 
    background-color: transparent;
     subcontrol-position: top left; /* position at the top left*/ 
     padding:2 13px;
 } 

QScrollArea { 
     border: 2px solid gray;
     background-color: lightgray;
     border-radius: 5px; 
 } 
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>277</width>
         <height>345</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="ColorWidget" name="fillColor">
            <property name="toolTip">
             <string>Custom color widget</string>
            </property>
            <property name="whatsThis">
             <string>You don't care</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
            </property>
            <property name="horizontalSpacing">
             <number>22</number>
            </property>
            <property name="verticalSpacing">
             <number>9</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Current Color:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="fillColorLabel">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel {
	border: 1px solid black;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Fill Enabled:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="shouldFillCheckbox">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="fontPage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>283</width>
     <height>351</height>
    </rect>
   </property>
   <attribute name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/tool-icons/text.png</normaloff>:/icons/resources/icons/tool-icons/text.png</iconset>
   </attribute>
   <attribute name="label">
    <string>Font</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollArea::title { 
    background-color: transparent;
     subcontrol-position: top left; /* position at the top left*/ 
     padding:2 13px;
 } 

QScrollArea { 
     border: 2px solid gray;
     background-color: lightgray;
     border-radius: 5px; 
 } 
</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>277</width>
         <height>345</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="FontWidget" name="fontWidget">
          <property name="toolTip">
           <string>Custom widget</string>
          </property>
          <property name="whatsThis">
           <string>You don't care</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="usersPage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>283</width>
     <height>351</height>
    </rect>
   </property>
   <attribute name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/app-icons/users.png</normaloff>:/icons/resources/icons/app-icons/users.png</iconset>
   </attribute>
   <attribute name="label">
    <string>Users</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea_5">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>277</width>
         <height>345</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <attribute name="headerDefaultSectionSize">
           <number>0</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="chatPage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>283</width>
     <height>351</height>
    </rect>
   </property>
   <attribute name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/app-icons/chat.png</normaloff>:/icons/resources/icons/app-icons/chat.png</iconset>
   </attribute>
   <attribute name="label">
    <string>Chat</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <widget class="CustomChatWidget" name="chatWidget">
      <property name="toolTip">
       <string>Custom widget</string>
      </property>
      <property name="whatsThis">
       <string>You don't care</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionCapSquare">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/app-icons/capsquare.png</normaloff>:/icons/resources/icons/app-icons/capsquare.png</iconset>
   </property>
   <property name="text">
    <string>capSquare</string>
   </property>
  </action>
  <action name="actionCapRound">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/app-icons/capround.png</normaloff>:/icons/resources/icons/app-icons/capround.png</iconset>
   </property>
   <property name="text">
    <string>capRound</string>
   </property>
  </action>
  <action name="actionCapFlat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/app-icons/capbutt.png</normaloff>:/icons/resources/icons/app-icons/capbutt.png</iconset>
   </property>
   <property name="text">
    <string>capFlat</string>
   </property>
  </action>
  <action name="actionJoinRound">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/app-icons/joinround.png</normaloff>:/icons/resources/icons/app-icons/joinround.png</iconset>
   </property>
   <property name="text">
    <string>joinRound</string>
   </property>
  </action>
  <action name="actionJoinMiter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/app-icons/joinmiter.png</normaloff>:/icons/resources/icons/app-icons/joinmiter.png</iconset>
   </property>
   <property name="text">
    <string>joinMiter</string>
   </property>
  </action>
  <action name="actionJoinBevel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/app-icons/joinbevel.png</normaloff>:/icons/resources/icons/app-icons/joinbevel.png</iconset>
   </property>
   <property name="text">
    <string>joinBevel</string>
   </property>
  </action>
  <action name="actionStrokeColor">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/tool-icons/color1.png</normaloff>:/icons/resources/icons/tool-icons/color1.png</iconset>
   </property>
   <property name="text">
    <string>strokeColor</string>
   </property>
  </action>
  <action name="actionFillColor">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/tool-icons/color1.png</normaloff>:/icons/resources/icons/tool-icons/color1.png</iconset>
   </property>
   <property name="text">
    <string>fillColor</string>
   </property>
  </action>
  <action name="actionShouldFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>shouldFill</string>
   </property>
  </action>
  <action name="actionShouldStroke">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>shouldStroke</string>
   </property>
  </action>
  <action name="actionWidthChange">
   <property name="text">
    <string>widthChange</string>
   </property>
  </action>
  <action name="actionDashSet">
   <property name="text">
    <string>dashSet</string>
   </property>
  </action>
  <action name="actionDashPhaseChange">
   <property name="text">
    <string>dashPhaseChange</string>
   </property>
  </action>
  <action name="actionFontStyleBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/app-icons/bold.png</normaloff>:/icons/resources/icons/app-icons/bold.png</iconset>
   </property>
   <property name="text">
    <string>fontStyleBold</string>
   </property>
  </action>
  <action name="actionFontStyleItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/app-icons/italic.png</normaloff>:/icons/resources/icons/app-icons/italic.png</iconset>
   </property>
   <property name="text">
    <string>fontStyleItalic</string>
   </property>
  </action>
  <action name="actionFontFamilySet">
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/icons/resources/icons/tool-icons/text.png</normaloff>:/icons/resources/icons/tool-icons/text.png</iconset>
   </property>
   <property name="text">
    <string>fontFamilySet</string>
   </property>
  </action>
  <action name="actionFontSizeSet">
   <property name="text">
    <string>fontSizeSet</string>
   </property>
  </action>
  <action name="actionChangeCharCounter">
   <property name="text">
    <string>changeCharCounter</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorWidget</class>
   <extends>QWidget</extends>
   <header>ColorWidget</header>
  </customwidget>
  <customwidget>
   <class>CustomChatWidget</class>
   <extends>QTabWidget</extends>
   <header>CustomChatWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DrawingPropertiesWidget</class>
   <extends>QWidget</extends>
   <header>DrawingPropertiesWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FontWidget</class>
   <extends>QWidget</extends>
   <header>FontWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StrokeStyleWidget</class>
   <extends>QWidget</extends>
   <header>StrokeStyleWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rsrc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>shouldFillCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionShouldFill</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>strokeColor</sender>
   <signal>colorChanged(QColor)</signal>
   <receiver>ToolBox</receiver>
   <slot>strokeColorChanged(QColor)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fillColor</sender>
   <signal>colorChanged(QColor)</signal>
   <receiver>ToolBox</receiver>
   <slot>fillColorChanged(QColor)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shouldStrokeCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionShouldStroke</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>strokeColorChanged(QColor)</slot>
  <slot>fillColorChanged(QColor)</slot>
 </slots>
</ui>
